src/test/java/org/cactoos/iterator/SyncIteratorTest.java

```diff
@@ -23,16 +23,22 @@
  */
 package org.cactoos.iterator;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
+import junit.framework.TestCase;
```

Comment:
@driver733 we don't use this. Use Hamcrest instead, like all other tests are doing